apps/vs-code-intellisense/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
```

Comment:
Question: same question as the eslint config, these compiler options are pretty similar to our global ones. The target, outDir, and lib are different. Can it use the global one instead?

Reply:
good point, updated

Reply:
Let's keep this package isolated, it's a unique workspace